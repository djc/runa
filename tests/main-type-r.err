main-type-r.rns [1.1]: main() return type must be i32
def main(name: &Str, args: &array[Str]) -> float:
^
