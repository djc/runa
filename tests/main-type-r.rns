def main(name: &str, args: &array[str]):
	print(x)
