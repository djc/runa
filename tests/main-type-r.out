tests/main-type-r.rns [1.1]: main() return type must be i32
def main(name: &str, args: &array[str]):
^
